### PR TITLE
fix: homepage/repository in package.json

### PR DESCRIPTION
**Pull request by domwoe**
Signed-off-by: Dominic Woerner <dominic.woerner@dfinity.org>

# Description
Homepage link in npm registry was broken and link to repository/package and issue tracking was missing.

# Checklist:

- [x] My changes follow the guidelines in [CONTRIBUTING.md](https://github.com/dfinity/agent-js/blob/main/CONTRIBUTING.md).
- [x] The title of this PR complies with [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/).
- [ ] I have edited the CHANGELOG accordingly.
- [ ] I have made corresponding changes to the documentation.

### CODE REVIEW

**Comment by domwoe** [comment]
@krpeacock Would you take a look? :)

File: packages/agent/package.json [hunk 1]
```diff
@@ -4,7 +4,15 @@
   "author": "DFINITY Stiftung <sdk@dfinity.org>",
   "license": "Apache-2.0",
   "description": "JavaScript and TypeScript library to interact with the Internet Computer",
-  "homepage": "https://sdk.dfinity.org/docs/index.html",
+  "homepage": "https://smartcontracts.org",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/dfinity/agent-js.git",
+    "directory": "packages/agent"
```
Comment (krpeacock) [inline packages/agent/package.json:11]: I don't think the directory config is correct - this package.json already lives inside packages/agent

**Review by krpeacock — CHANGES_REQUESTED** [review]

**Comment by krpeacock** [comment]
Thanks for opening this! One note about the repository/directory configs before we accept this

File: packages/agent/package.json (hunk 1 above)
Comment (domwoe) [reply packages/agent/package.json:11]: As far as I understand, the `directory` refers just to the repo given by the `url` above, i.e. the base repo, and has nothing to do with the `package.json`.

Comment (domwoe) [reply packages/agent/package.json:11]: @krpeacock

**Review by krpeacock — APPROVED** [review]

**Pull request merged**